addons/spreadsheet/static/tests/global_filters/global_filters_chart_test.js

```diff
@@ -107,6 +107,11 @@ QUnit.module("spreadsheet > Global filters chart", {}, () => {
             type: "date",
         };
         assert.deepEqual(model.getters.getChartFieldMatch(chartId)[filter.id], matching);
+        assert.deepEqual(model.getters.getChartDataSource(chartId)._customDomain, [
```

Comment:
```suggestion
        assert.deepEqual(model.getters.getChartDataSource(chartId).getComputedDomain(), [
```
You can use the public API instead of the private attribute :)